### DISCUSSION TITLE
Open linked files in PHPStorm

### DISCUSSION THREAD

**Discussion by raveren** [spatie/laravel-ignition#143] [Q&A]
Not for the lack of trying but I was never able to make the `phpstorm://` protocol work on neither OS I was on, although I am on windows now and can vouch there is no way to open the IDE via mentioned protocol.

Relevant, unclosed, ancient issue:

https://youtrack.jetbrains.com/issue/IDEA-65879/idea-protocol-handler-to-open-files-directly-from-a-URL

Finally, however, Jetbrains made their plugin meant to do just that - and it actually works at last:

https://plugins.jetbrains.com/plugin/19991-ide-remote-control

The format for the link is this:

```
http://localhost:63342/api/file/%file:%line
```

Can we have a separate option in the editor list for this? Or can we have an option to add a custom format? Because as it is now, Ignition provides **no way** to open PHPStorm from the error pages....

**Comment by raveren** [comment]
Okay, so how do **you** open your IDE at the relevant file:line from an error page?

1. I use Visual Studio or another IDE (not PHPStorm)
2. I found a way to use the `phpstorm://` protocol!
3. I don't

**Comment by lk77** [comment]
Same thing, i was never able to make it work on windows and linux

**Reply by parallels999** [reply]
Did you try `phpstorm-remote`?
https://github.com/spatie/ignition/blob/20eab2791feb492e5afe0002cd093155df80fcaa/src/Config/IgnitionConfig.php#L178-L181
